addons/l10n_cl/views/report_invoice.xml

```diff
@@ -1,65 +1,247 @@
+            <div class="col-6">
+                <strong>
+                    <span t-att-style="'color: %s;' % o.company_id.secondary_color">Date:</span>
+                </strong>
+                <span t-esc="report_date" t-options='{"widget": "date"}'/>
+                <br/>
+
+                <strong>Customer:</strong>
+                <span t-field="o.partner_id.name"/>
+                <br/>
+
+                <t t-if="o.partner_id.vat and o.partner_id.l10n_latam_identification_type_id">
+                    <strong>
+                        <t t-esc="o.partner_id.l10n_latam_identification_type_id.name or o.company_id.country_id.vat_label" id="inv_tax_id_label"/>:
+                    </strong>
+                    <span t-esc="o.partner_id.vat"/>
+                    <br/>
+                </t>
+                <strong>GIRO:</strong>
+                <span t-esc="o.partner_id.industry_id.name or ''"/>
+                <br/>
+            </div>
+            <div class="col-6">
+                <strong>Due Date:</strong>
+                <span t-esc="o.invoice_date_due" t-options='{"widget": "date"}'/>
+                <br/>
+                <strong>Address:</strong>
+                <span t-field="o.partner_id"
+                      t-options="{&quot;widget&quot;: &quot;contact&quot;, &quot;fields&quot;: [&quot;address&quot;], &quot;no_marker&quot;: true, &quot;no_tag_br&quot;: True}"/>
+
+                <strong>Payment Terms:</strong>
+                <span t-esc="o.invoice_payment_term_id.name or ''"/>
+
+
+                <t t-if="o.invoice_incoterm_id">
+                    <br/>
+                    <strong>Incoterm:</strong>
+                    <span t-field="o.invoice_incoterm_id.name"/>
+                </t>
+
+            </div>
+        </div>
+        <div id="references" class="row">
+            <div name="references" class="col-12 text-center"/>
+        </div>
+    </template>
+
+    <template id="custom_footer">
+        <div name="footer_left_column" class="col-8 text-center"/>
+    </template>
+
     <template id="report_invoice_document" inherit_id="account.report_invoice_document" primary="True">
 
-        <xpath expr="//h2" position="attributes">
-            <attribute name="t-if">not o.l10n_latam_document_type_id.report_name</attribute>
-        </xpath>
+        <t t-set="o" position="after">
+            <t t-set="custom_header" t-value="'l10n_cl.custom_header'"/>
+            <t t-set="report_date" t-value="o.invoice_date"/>
+            <t t-set="report_number" t-value="int(o.l10n_latam_document_number)"/>
+            <t t-set="pre_printed_report" t-value="report_type == 'pdf'"/>
+            <t t-set="report_name" t-value="o.l10n_latam_document_type_id.name"/>
+            <t t-set="header_address" t-value="o.company_id.partner_id"/>
+            <t t-set="is_tax" t-value="o.l10n_latam_document_type_id.code not in ['39', '41', '110', '111', '112', '34']"/>
```

Comment:
Should be set in the main report, not here.

Reply:
Can you check

Reply:
in the custom_header you mean?

Reply:
@jco-odoo moved to the custom_header: 4d1f10a546e5de417bae9b2a12bd6bc3de92418c